src/lib-voyager.tsx

```diff
@@ -169,6 +169,18 @@ export class Voyager {
     return spec;
   }
 
+  /**
+   *
+   * Gets the current bookmarked vega-lite specs.
+   *
+   * @returns {string[]}
+   *
+   * @memberof Voyager
+   */
+  public getBookmarkedSpecs(): string[] {
+    return this.store.getState().persistent.bookmark.list;
```

Comment:
I recommend you use the `selectBookmark` selector rather than `this.store.getState().persistent.bookmark` in case we ever change the shape of the state in the future.